llama_index/indices/knowledge_graph/base.py

```diff
@@ -104,13 +125,13 @@ def _parse_triplet_response(response: str) -> List[Tuple[str, str, str]]:
     def _build_index_from_nodes(self, nodes: Sequence[Node]) -> KG:
         """Build the index from nodes."""
         # do simple concatenation
-        index_struct = KG(table={})
+        index_struct = self.index_struct_cls(table={})
```

Comment:
nit: Is `table={}` needed? It should hopefully already be defaulting to a dict

Reply:
True, KG already handles it properly, it's removed now.